src/components/VideoList/VideoList.spec.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { render } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
```

Comment:
If you try to share [this link](https://wz-react-certification-2020-alexismundu.netlify.app/video/Po3VwR_NNGk) of you app, people will get a `Page Not Found` error, take a look at [this post](https://dev.to/rajeshroyal/page-not-found-error-on-netlify-reactjs-react-router-solved-43oa) to see a possible solution